example/fields.js

```diff
@@ -26,7 +26,8 @@ module.exports = {
     placeholder: "Student or professional at ...",
     validator: {
       error: "Please let us know what your occupation is."
```

Comment:
Because we're now also putting a restriction on word limit, let's change this to "Please let us know what your occupation is (in 30 words or fewer)."